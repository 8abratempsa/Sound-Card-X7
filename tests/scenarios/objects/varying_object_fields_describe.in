octosql "SELECT * FROM fixtures/test.json" --output stream_native --describe
