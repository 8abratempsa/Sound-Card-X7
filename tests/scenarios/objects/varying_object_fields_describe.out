{+0001-01-01T00:00:00Z| 'test.field1', 'String', false |}
{+0001-01-01T00:00:00Z| 'test.field2', 'Float', false |}
{+0001-01-01T00:00:00Z| 'test.field3', '{field4: NULL | String; field5: Float | String; field6: NULL | String}', false |}
